charts/redpanda/values.yaml

```diff
@@ -344,6 +354,12 @@ serviceAccount:
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
   name: ""
+# Role Based Access Control
+rbac:
+  # enable this for features that need extra priveleges
+  enabled: false
```

Comment:
Should this be enabled by default? I feel that it should as it is required to enable other features out of the box. For instance, we enable external access out of the box, but if we want it to work then they would need to flip this flag (eventually). The same applies to rack awareness, in that they would need to enable rack awareness and also rbac in order to get just rack awareness.

Another option may be to do a check that doesn't allow users to enable either rack awareness or external access without also enabling rbac. I this would work to ensure clusters work as expected, but at the same time making rbac be enabled automatically when these other features are enabled would be a better default.

Reply:
The principal of least privilege suggests that of a privilege is not needed it should not be added. Unless rack awareness is enabled, there's no need for the container to be able to get the Node.

Reply:
That's true for rack awareness. I guess another approach would be to enable rbac by default in [another incoming PR](https://github.com/redpanda-data/helm-charts/pull/224) that fixes external access by making use of this (given that external access is enabled by default.